### ISSUE TITLE
Reproduction on LIBERO

### ISSUE THREAD

**Issue by SMLee0407** [OpenHelix-Team/VLA-Adapter#46]
Thank you for sharing your great work.

I am currently working on reproducing the fine-tuning results on the LIBERO-Goal suite. 
While the expected success rate is approximately 98%, I achieved a 94% success rate with my current setup.

I followed the configuration provided in the "How to Train on Larger VRAM GPUs" section, except I adjusted the max_steps.
My current configuration for libero_goal:

GPU: 1 (Single GPU environment for both training and evaluation)
Batch size: 8
Gradient accumulation steps: 2 (Effective Batch Size: 16)
Training steps: 50,000 steps
Use pro version: True

I noticed in a previous [issue #35](https://github.com/OpenHelix-Team/VLA-Adapter/issues/35) that for a 4-GPU setup (which mentions 16 batch per gpu configuration), the recommended steps were:

Long: 60,000 steps
Object: 5,000 steps
Goal: 50,000 steps
Spatial: 20,000 steps

Since my effective batch size and hardware setup differ from the one mentioned above, I have been unable to reach the reported performance.

Could you please share the recommended training steps for each benchmark (Long, Object, Goal, Spatial) specifically for the "How to Train on Larger VRAM GPUs" (Single GPU, Batch 8, Grad Acc 2) setting to obtain reproducible results ? 

Thank you for your support.
